tests/lib/hardware/libvirt.py

```diff
@@ -253,8 +253,8 @@ def __init__(self):
         super().__init__()
         self._network = self._create_network()
         if not self._network:
-            raise Exception(f'Can not get libvirt network '
-                            '{config.PROVIDER_LIBVIRT_NETWORK_RANGE}')
+            raise Exception('Can not get libvirt network %s' %
```

Comment:
Why that? Just add a `f` in front of the second line

Reply:
Because it's split over two lines anyway I figured a substitution was just as neat.